### ISSUE TITLE
SageNB: Notebook Settings page

### ISSUE THREAD

**Issue by sageb0t** [sageb0t/testsage#4551]
Imported from <a href="http://trac.sagemath.org/sage_trac/ticket/4551">trac #4551</a>.

Now uses Farbtastic as a color chooser.  Rebased against #7196 + #7158 v2.  Apply only this patch.

Reported by: TimothyClemans

Component: notebook

CC: wjp,, timdumol,, was

Report Upstream: 

Authors: Timothy Clemans, Mitesh Patel

Dependencies: 

Work issues: 

Reviewers: was

Merged in: 

Attachments: <ol><li><a href="http://trac.sagemath.org/sage_trac/attachment/ticket/4551/sage-4551.patch">sage-4551.patch: </a> by TimothyClemans at 20081203T15:19:39</li><li><a href="http://trac.sagemath.org/sage_trac/attachment/ticket/4551/sage-4551_2.patch">sage-4551_2.patch: </a> by TimothyClemans at 20081203T15:20:29</li><li><a href="http://trac.sagemath.org/sage_trac/attachment/ticket/4551/trac_4551.patch">trac_4551.patch: first two patches folded into one, rebased on 4.1.2.alpha0 + #4550</a> by ddrake at 20090905T01:31:41</li><li><a href="http://trac.sagemath.org/sage_trac/attachment/ticket/4551/trac_4550+4551-settings_page.patch">trac_4550+4551-settings_page.patch: Rebased for sagenb #6983 + #7110 (including #5447) + #7158 + #4550.</a> by mpatel at 20091010T07:25:49</li><li><a href="http://trac.sagemath.org/sage_trac/attachment/ticket/4551/trac_4551-settings_page_v2.patch">trac_4551-settings_page_v2.patch: SageNB: settings page for all server options.  Depends on #7110, #7158.  Apply only this patch.</a> by mpatel at 20091010T13:08:10</li><li><a href="http://trac.sagemath.org/sage_trac/attachment/ticket/4551/trac_4551-settings_page_v3.patch">trac_4551-settings_page_v3.patch: Now uses Farbtastic as a color chooser.  Rebased against #7196 + #7158 v2.  Apply only this patch.</a> by mpatel at 20091014T10:56:42</li></ol>
